Bottom: f4b05febc9a73ebc11702e488a16670dd0c0f9da
Top:    bf0330039ddb7492c9d32da5cfea387f7b606a73
Author: Mark de Wever <koraq@xs4all.nl>
Date:   2021-08-12 20:47:22 +0200

Refresh of add-a-calculator-controller

---

diff --git a/src/modules/calculator/CMakeLists.txt b/src/modules/calculator/CMakeLists.txt
index b41cf3a..e3e7585 100644
--- a/src/modules/calculator/CMakeLists.txt
+++ b/src/modules/calculator/CMakeLists.txt
@@ -17,7 +17,9 @@ add_module(calculator.value value.cpp)
 add_dependencies(calculator.model.pcm
 	calculator.value.pcm
 )
+add_dependencies(calculator.controller.pcm
+	calculator.model.pcm
+)
 add_dependencies(calculator
 	calculator.controller.pcm
-	calculator.model.pcm
 )
